examples/resp-headings/README.md

```diff
@@ -0,0 +1,41 @@
+# resp-headings
+
+This plugin adds responsive heading components to Tailwind.
+
+The limitation to this plugin is that you cannot use `@apply .heading-key`. Tailwind explicitly prevents the use of `@apply` with "complex" components with media queries.
+
+## Usage
+
+```js
+theme: {
+  headings: theme => ({
+    '24': {
+      sizes: {
+        default: theme('fontSize.20'),
+        md: theme('fontSize.24')
+      },
+      lineHeight: theme('lineHeight.snug'),
+      weight: theme('fontWeight.bold')
+    },
+  })
+}
+```
```

Comment:
Add example plugin array for consistency w/ the other plugins...
```
plugins: [
  require('@viget/tailwindcss-plugins/headings'),
],
```

Reply:
@jeremyfrank - this is included as an example and isn't actually baked into the npm module. So, this `require` wouldn't work, I don't think. That's why I left it out of the docs.

Maybe update it to this instead?
```
plugins: [
  require('./config/tailwind/plugins/resp-headings'),
],
```

... assuming they're baking it into their project

Reply:
Oops you're right! What you added is good. 👍